index.js

```diff
@@ -39,20 +39,34 @@ client.settings = new PersistentCollection({name: "settings"});
 
 const init = async () => {
 
-  // Here we load **commands** into memory, as a collection, so they're accessible
-  // here and everywhere else.
-  const cmdFiles = await readdir("./commands/");
-  client.log("log", `Loading a total of ${cmdFiles.length} commands.`);
-  cmdFiles.forEach(f => {
+  /*
+  Here we load **commands** into memory, as a collection,
+  so they're accessible here and everywhere else.
+  Useing the pathwalker function out of functions.js.
+  */
+  const cmdFiles = await client.pathwalker("./commands/");
+  client.log("log", `Loading a total of ${cmdFiles[1].length} commands out of ${cmdFiles[0].length} folders.`);
+  // for each CMD
+  cmdFiles[1].forEach(f => {
     try {
-      const props = require(`./commands/${f}`);
+      // try each Path
+      cmdFiles[0].forEach(d =>{
+        try {
+          client.props = require(`./commands/${d}/${f}`);
```

Comment:
No. `client.props` is global, that's absolutely horrifyingly stupid. `const props` must be a LOCALLY SCOPED VAR. putting it on client makes no goddamn sense.